source/iot/Mqtt5Client.cpp

```diff
@@ -217,6 +217,25 @@ namespace Aws
             return result;
         }
 
+        Mqtt5ClientBuilder *Mqtt5ClientBuilder::NewMqtt5ClientBuilderWithCustomAuthorizerWebsocket(
+            const Crt::String hostName,
+            const Mqtt5CustomAuthConfig &customAuthConfig,
+            const WebsocketConfig &config,
+            Crt::Allocator *allocator) noexcept
+        {
+            Mqtt5ClientBuilder *result = new Mqtt5ClientBuilder(allocator);
+            result->m_tlsConnectionOptions = Crt::Io::TlsContextOptions::InitDefaultClient();
+            if (!result->m_tlsConnectionOptions)
```

Comment:
The m_tlsConnectionOptions validation was incorrect. Please checkout the fix here. Sorry for the mistake. 
https://github.com/awslabs/aws-crt-cpp/pull/445/files#diff-318e7df61fb2407a67e3297977004fe83850004bf7698e5b580379bfcbfede01L126